.travis.yml

```diff
@@ -1,36 +1,49 @@
-os: linux
+os:
+  - linux
+  - osx
+  - windows
 dist: trusty
-language: python
-python: 3.6
-
-env:
- - TRAVIS_NODE_VERSION=12
+language: shell 
+cache: pip
 
 services:
  - mongodb
 
-matrix:
+env: 
+  jobs:   
+   - MAKE_COMMAND=test
+   - MAKE_COMMAND=integration-test
+   - MAKE_COMMAND=package
```

Comment:
Pretty sure splitting this into jobs will just take way more time, as each job would have to do the long `make build` step themselves.

Reply:
Yeah this was the biggest reason I stopped trying to make the multiple job stuff work. I tried to non-splittily make the build faster https://github.com/ActivityWatch/activitywatch/tree/dev/less-breaking-travis-improvements by using caches, and I think it's a better direction.